docs: intro sentence for date-dropdown (#1605)
siemens/ix: packages/documentation/docs/controls/pagination.md

# Pagination

<Tags />
<!-- introduction start -->

> Paginations allow users to navigate between pages of content when it is split for performance reasons.

Reviewer: ```suggestion
Paginations allow users to navigate between pages of content when it's split for performance reasons.
```
